Gemfile.lock

```diff
@@ -1,143 +0,0 @@
-GEM
```

Comment:
@chris-rock any idea why Gemfile.lock was commited in this repo? Do we need it this way?